picard/ui/mainwindow.py

```diff
@@ -174,6 +176,19 @@ def keyPressEvent(self, event):
     def show(self):
         self.restoreWindowState()
         super().show()
+        for key in PROGRAM_UPDATE_LEVELS:
+            if PROGRAM_UPDATE_LEVELS[key]['level'] == config.setting["update_level"]:
```

Comment:
Just a minor thing, use single quotes around `update_level` for consistency

Reply:
Will do.  I have to go out to run some errands right now, but will fix that (along with the log translations) when I get back.